apps/web/src/components/formatter/locale-date.tsx

```diff
@@ -10,7 +10,7 @@ export const LocaleDate = ({ className, date, ...props }: LocaleDateProps) => {
   const [localeDate, setLocaleDate] = useState(() => new Date(date).toISOString());
 
   useEffect(() => {
-    setLocaleDate(new Date(date).toLocaleString());
```

Comment:
I would assume we want to include the time since it could be a key differentiator in the future.

Reply:
Sure. I'll reverse it back